sdks/python/apache_beam/runners/interactive/caching/datatype_inference.py

```diff
@@ -0,0 +1,109 @@
+#
```

Comment:
I think this module is better placed under `apache_beam/testing/`.
This signals to others that it is for use in tests only, and generally available for reuse in other tests (not just interactive runner).